crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -202,9 +202,6 @@ where
 #[derive(Resource)]
 struct AnyPrepassPluginLoaded;
 
-#[cfg(not(feature = "meshlet"))]
-type PreviousViewFilter = (With<Camera3d>, With<MotionVectorPrepass>);
-#[cfg(feature = "meshlet")]
 type PreviousViewFilter = Or<(With<Camera3d>, With<ShadowView>)>;
```

Comment:
We can just inline this and remove the type def now.

Reply:
I only added the typedef in the first place to deal with the meshlet feature. In hindsight, not actually all that necessary, this probably costs very little perf (unlike the previous instance system).

Reply:
Done.